packages/rs-drive/tests/supporting_files/contract/withdrawals/withdrawals.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
Is this contract modified? We already have contracts in crates and they are re-exported from DPP